src/Dockerfile_base

```diff
@@ -15,13 +15,12 @@ apt-get -y install build-essential vim git python3-pip \
 
 # 2. convenient symlinks; NFS mount folder
 RUN ln -s /usr/bin/python3 /usr/bin/python && \
-ln -s /usr/bin/pip3 /usr/bin/pip && \
```

Comment:
Hm, I prefer to have `pip3`/`python3` aliased to `pip`/`python`, since we shouldn't be configuring Python 2.7 environments anymore.

Was this causing issues with legacy software?

Reply:
I saw error:
```
Step 4/20 : RUN ln -s /usr/bin/python3 /usr/bin/python && ln -s /usr/bin/pip3 /usr/bin/pip && mkdir -p /mnt/nfs
 ---> Running in 75f7675a0809
ln: failed to create symbolic link '/usr/bin/pip': File exists
```
and `/usr/bin/pip --version` shows that it is from `pip 20.0.2 from /usr/lib/python3/dist-packages/pip (python 3.8)`.

So I think `apt-get` already does that aliasing for us in recent changes.

Reply:
Awesome, glad that the distros have finally moved into the 21st century :-)